src/phase0/state_transition/mod.rs

```diff
@@ -1150,8 +1148,8 @@ pub fn slash_validator<
         state.validators[slashed_index].withdrawable_epoch,
         epoch + context.epochs_per_slashings_vector as u64,
     );
-    state.slashings[epoch as usize % EPOCHS_PER_SLASHINGS_VECTOR as usize] +=
-        state.validators[slashed_index].effective_balance;
+    let slashing_index = epoch as usize % EPOCHS_PER_SLASHINGS_VECTOR;
+    state.slashings[slashing_index] += state.validators[slashed_index].effective_balance;
```

Comment:
```suggestion
    let slashings_index = epoch as usize % EPOCHS_PER_SLASHINGS_VECTOR;
    state.slashings[slashings_index] += state.validators[slashed_index].effective_balance;
```

just to make it clear this is the index into `slashings` and not related to the slashed validator

Reply:
ok